editors/code/src/ctx.ts

```diff
@@ -73,11 +73,11 @@ export class Ctx {
         }
     }
 
-    get subscriptions(): { dispose(): any }[] {
+    get subscriptions(): { dispose(): unknown }[] {
         return this.extCtx.subscriptions;
     }
 
-    pushCleanup(d: { dispose(): any }) {
+    pushCleanup(d: { dispose(): unknown }) {
```

Comment:
Similarly, this wants to be a `: void`?

Reply:
In general, yes you are right, now when I am looking at it, this seems reasonable